# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../libgli/cc.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
diff -pruN libgli-0.4_orig/src/Makefile libgli-0.4/src/Makefile
--- libgli-0.4_orig/src/Makefile	2002-12-16 08:24:07.000000000 +0100
+++ libgli-0.4/src/Makefile	2004-01-17 11:23:16.000000000 +0100
@@ -2,16 +2,18 @@ CFLAGS = -pedantic -ansi -Wall -I./inclu
 LFLAGS = -ldl -shared
 C++ = g++
 
-all: libgli.so
+all: libgli.so libgli.a
 
 uninstall:
 	rm -f $(prefix)/lib/libgli-$(VERSION).so
 	rm -f $(prefix)/lib/libgli.so
+	rm -f $(prefix)/lib/libgli.a
 	rm -f $(prefix)/include/gli*.h
 
-install: libgli.so
+install: libgli.so libgli.a
 	cp libgli.so $(prefix)/lib/libgli-$(VERSION).so
 	ln -sf $(prefix)/lib/libgli-$(VERSION).so $(prefix)/lib/libgli.so
+	cp libgli.a $(prefix)/lib/libgli.a
 	cp include/gli*.h $(prefix)/include
 
 gli_button.o: gli_button.cpp
@@ -58,6 +60,13 @@ libgli.so: gli_button.o gli_frame.o gli_
 	gli_texture_tga.o gli_editbox.o gli_image.o gli_progress.o
 	$(C++) $(CFLAGS) gli*.o -o libgli.so $(LFLAGS)
 
+libgli.a: gli_button.o gli_frame.o gli_glf.o gli_interface.o gli_object.o \
+	gli_text.o gli_texture.o gli_window.o gli_texture_rgb.o \
+	gli_texture_tga.o gli_editbox.o gli_image.o gli_progress.o
+	ar rcs libgli.a gli*.o
+	ranlib libgli.a
+
 clean:
 	rm -f *.o
 	rm -f libgli.so
+	rm -f libgli.a
